data/companies.json

```diff
@@ -13,6 +13,34 @@
       "youtube": null
     }
   },
+  {
+    "name": "CryptoGen Nepal",
+    "establishedAt": 1553609490,
+    "location": "Naaxl, Kathmandu",
+    "website": "https://cryptogennepal.com/",
+    "socials": {
+      "facebook": "https://www.facebook.com/CryptoGenNepal",
+      "github": "https://github.com/cryptogennepal",
+      "instagram": "https://instagram.com/CryptoGenNepal",
+      "linkedin": "https://np.linkedin.com/company/cryptogennepal",
```

Comment:
Can we have the `https://linkedin.com` domain?
We are trying to maintain standards

Reply:
Done